lib/surface/components/events.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Surface.Components.Events do
+  @moduledoc false
```

Comment:
I believe we should make this module public, so developer/user can use it in their own components. WDYT?

We could even group the event by type, like in the [doc](https://hexdocs.pm/phoenix_live_view/bindings.html), and let the user choose which type of events, he would like to import:

```elixir
use Surface.Components.Events, only: [:click, :focus_blur, :form, :key] 
```

Reply:
The thought had crossed my mind, but not sure if the benefits outweigh the cost of increasing the complexity of the Surface API.